### PR TITLE
style controlled arena-hard

### PR DESCRIPTION
**Pull request by jveronvialard**
Style control for Arena-Hard. Adapted from https://github.com/lmarena/arena-hard-auto/blob/main/show_result.py with pre-computed normalization constants and regression coefficients for the model style features. Also replace point estimate by bootstrap mean.

## Summary by CodeRabbit

* **New Features**
  * Style-control added to arena metrics with per-example style features, category-aware aggregation, and option to apply normalization/coefficient tables.
  * Elo estimation now supports control features, integrated bootstrapping, and more stable win-rate computation.

* **Tests**
  * Expanded tests for style-control workflows, normalization/coefficient options, and edge cases; updated evaluation fixtures and expected outputs.

* **Chores**
  * Added statsmodels dependency and minor prompt-category mapping for judge setup.

### CODE REVIEW

**Review by Kipok — COMMENTED** [review]
no concerns from my side, but I didn't check logic carefully. Please update the failing test (golden metrics needs to be recomputed) and also maybe we should update the docs in https://github.com/NVIDIA-NeMo/Skills/blob/main/docs/evaluation/other-benchmarks.md to have new expected results (if significantly changed) and maybe a short description of new options)

File: nemo_skills/evaluation/metrics/arena_metrics.py [hunk 1]
```diff
@@ -40,6 +71,70 @@ def get_incorrect_sample(self, prediction: dict) -> dict:
...
+        # model features are normalized against baseline features
+        normalized_feature_tensor = np.zeros_like(answer_feature_tensor, dtype=float)
+        normalized_feature_tensor[0] = (answer_feature_tensor[0] - baseline_feature_tensor[0]) / (
+            answer_feature_tensor[0] + baseline_feature_tensor[0]
+        )
+        model_md_density = answer_feature_tensor[1:] / (answer_feature_tensor[0] + 1)
+        baseline_md_density = baseline_feature_tensor[1:] / (baseline_feature_tensor[0] + 1)
+        normalized_feature_tensor[1:] = (model_md_density - baseline_md_density) / (
+            model_md_density + baseline_md_density + 1
+        )
```
[the remark this answers is not in the document]
Comment (jveronvialard) [reply nemo_skills/evaluation/metrics/arena_metrics.py:150]: baseline is never empty

File: nemo_skills/evaluation/metrics/arena_metrics.py [hunk 2]
```diff
@@ -79,39 +179,139 @@ def update(self, predictions):
...
+                control.
+            category_style_control_normalization_factors (dict | None): Per-category mean/std for
+                style features. If None, uses empirical mean/std from the current data.
+            category_style_control_coefs (dict | None): Per-category fixed regression coefficients.
+                If None, fits coefficients from the current data.
+        """
         from nemo_skills.evaluation.evaluator.arena import get_aggregate_score
 
-        metrics_dict = {}
+        unique_categories = sorted(set(self.categories))
```
[the remark this answers is not in the document]
Comment (jveronvialard) [reply nemo_skills/evaluation/metrics/arena_metrics.py:234]: No mixed data

**Comment by jveronvialard** [comment]
Adding this feature to nemo-gym instead
